package.json

```diff
@@ -7,9 +7,12 @@
     "@testing-library/react": "13.3.0",
     "@testing-library/user-event": "13.5.0",
     "axios": "0.27.2",
+    "country-flag-icons": "1.5.5",
```

Comment:
You installed this library, but I don't see you using it anywhere. Remove it and run `npm install` again.

Reply:
Okay, i will remove it